.github/workflows/build.yml

```
@@ -120,7 +120,12 @@ jobs:
         CIBW_ENVIRONMENT_MACOS: "MACOSX_DEPLOYMENT_TARGET=13 SYSTEM_VERSION_COMPAT=0"
         CIBW_BEFORE_ALL_WINDOWS: python -m pip install --upgrade pip
         CIBW_TEST_REQUIRES: pytest
-        CIBW_TEST_COMMAND: py.test -v {project}/tests
+        # we have to rename the `chiabip158` source or our stupid tests import the
+        # source, which is missing the .so file. You might think "why not just
+        # change the directory?" That's a great question and I've tried many
+        # things to do this and nothing works. Debugging in cibuildwheel is
+        # next to impossible.
+        CIBW_TEST_COMMAND: mv {project}/chiabip158 {project}/DO_NOT_IMPORT && py.test -v {project}/tests
```

and this is one of the various reasons that `src/myprojectname/` is a useful layout.  just to help understand some of the complexity, `python -m pytest` and `pytest` even behave differently around import paths etc...  `\o/`

trio does this (or did).  blockchain does it...

https://github.com/Chia-Network/chia-blockchain/blob/8463909e8ffb97d87f2cfadaa35bab52814685cf/.github/workflows/test-single.yml#L248-L250
```yaml
      - name: Move chia/ so we test the installed code
        run: |
          mv chia/ notchia/
```

```suggestion
        # rename source to avoid importing it and force testing the installed wheel
        CIBW_TEST_COMMAND: mv {project}/chiabip158 {project}/DO_NOT_IMPORT && py.test -v {project}/tests
```